.github/workflows/publish.yaml

```diff
@@ -16,21 +16,22 @@ jobs:
         with:
           lfs: true
       -
-        name: Read chrome-version.txt
-        id: package
-        uses: juliangruber/read-file-action@v1
-        with:
-          path: ./chrome-version.txt
-      -
-        name: Set env var with chrome-version
+        name: Set browser version
         id: set-browser-version
-        run: echo "CHROME_VERSION=${{ steps.package.outputs.content }}" >> $GITHUB_ENV
+        run: |
+          IFS='/-'; PARTS=($GITHUB_REF); unset IFS;
+          echo "BROWSER=${PARTS[2]}" >> $GITHUB_ENV
+          echo "BROWSER_VERSION=${PARTS[3]}" >> $GITHUB_ENV
+          if [[ "${PARTS[2]}" != "chrome" && "${PARTS[2]}" != "brave" ]]; then
+            echo "Exiting, invalid browser in tag '${PARTS[2]}', must be either 'chrome' or 'brave'"
+            exit 1
+          fi
       -
         name: Prepare
         id: prep
         run: |
           DOCKER_IMAGE=webrecorder/browsertrix-browser-base
-          TAGS="${DOCKER_IMAGE}:chrome-${{ env.CHROME_VERSION }}"
+          TAGS="${DOCKER_IMAGE}:${{ env.BROWSER }}-${{ env.BROWSER_VERSION }},${DOCKER_IMAGE}:${{ env.BROWSER }}"
```

Comment:
I like `${DOCKER_IMAGE}:${{ env.BROWSER }}` as an alternative to having a `:latest` for each browser. Nice!